src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkResolver.java

```diff
@@ -28,6 +30,7 @@
    *
    * @return Path to Google Cloud SDK or null
```

Comment:
Actually, it makes sense that a resolver can't resolve the location of the SDK and returns null. In that case, the next resolver is used.
Removing the annotation and keeping the sdkPath null check on validate().